File: 백준/Bronze/18108. 1998년생인 내가 태국에서는 2541년생？！/1998년생인 내가 태국에서는 2541년생？！.js
const fs = require('fs');

// trim()을 사용하여 입력값 끝의 줄바꿈 문자 등을 제거합니다.
const input = fs.readFileSync('/dev/stdin').toString().trim();

// 결과 출력
const num1 = Number(input);
console.log(num1 - 543);

File: 백준/Bronze/18108. 1998년생인 내가 태국에서는 2541년생？！/1998년생인 내가 태국에서는 2541년생？！.rs
use std::io;
fn main() {
    let mut input = String::new();
    
    io::stdin()
        .read_line(&mut input)
        .expect("입력을 읽지 못했습니다.");
    
   let number: i32 = input
        .trim()
        .parse()
        .expect("숫자가 아닙니다.");
 
    let result = number - 543;
    
    println!("{}",result);
}